Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.py

```diff
@@ -1962,6 +2043,155 @@ def cancel_containment_command(client: Client, args: Dict[str, Any]) -> CommandR
+
+
+def create_static_host_set_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    host_set_name = args.get('host_set_name')
+    hosts_ids = argToList(args.get('hosts_ids'))
+
+    if not hosts_ids:
+        raise ValueError('Please provide at least one Host ID.')
+    elif not host_set_name:
+        raise ValueError('Host Set name is required.')
+    data = None
+    try:
+        response = client.create_static_host_set_request(host_set_name, hosts_ids)
+        data = response['data']
+        message = f'Static Host Set {host_set_name} was created successfully.'
+    except Exception as e:
+        if '409' in str(e):
+            message = "Another host set has that name, please choose a different one."
+        else:
+            message = "Creating Host Set failed, check if you have the necessary permissions."
+
+    return CommandResults(
+        outputs_prefix='FireEyeHX.HostSets',
+        outputs_key_field="_id",
+        outputs=data,
+        readable_output=message
+    )
+
+
+def update_static_host_set_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    host_set_id = args.get('host_set_id')
+    host_set_name = args.get('host_set_name')
+    add_host_ids = argToList(args.get('add_host_ids'))
+    remove_host_ids = argToList(args.get('remove_host_ids'))
+
+    if not host_set_id:
+        raise ValueError('Host ID is required.')
+    elif not host_set_name:
+        raise ValueError('Host Set name is required.')
+    data = None
+    try:
+        response = client.update_static_host_set_request(host_set_name, host_set_id, add_host_ids, remove_host_ids)
+        data = response['data']
+        message = f'Static Host Set {host_set_name} was updated successfully.'
+    except Exception as e:
+        if '409' in str(e):
+            message = 'Another host set has that name, please choose a different one.'
+        elif '404' in str(e):
+            message = 'Host set was not found.'
+        else:
+            message = 'Updating Host Set failed, check if you have the necessary permissions.'
+
+    return CommandResults(
+        outputs_prefix='FireEyeHX.HostSets',
+        outputs_key_field="_id",
+        outputs=data,
+        readable_output=message
+    )
+
+
+def create_dynamic_host_set_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    host_set_name = args.get('host_set_name')
+    query = args.get('query')
+    query_key = args.get('query_key')
+    query_value = args.get('query_value')
+    query_operator = args.get('query_operator')
+
+    if not host_set_name:
+        raise ValueError('Host Set name is required.')
+    elif query and (query_key or query_value or query_operator):
+        raise ValueError('Cannot use free text query with other query operators, Please use one.')
+    elif not (query_key and query_value and query_operator) and not query:
+        raise ValueError('Please provide a free text query, or add all of the query operators toghether.')
```

Comment:
you need to check here that all of query_key, query_value and query_operator are used.